resource/ProgramAB/Dr.Who/config/DefaultPredicates.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
normally, these are txt files...  how did we end up with xml for these?  I'd be surprised if these files work with ProgramAB.

Reply:
it is 2 years old and was a draft in progress, not sure why I had that xml. Anyway I had pushed it as a txt long ago. Dr Who has a txt predicate now. I am closing this draft